lib/engine/game/g_1858/game.rb

```diff
@@ -461,6 +472,33 @@ def rust_phase4_trains!(purchased_train)
           rust_trains!(purchased_train, purchased_train.owner)
         end
 
+        def convert!(corporation, quiet: false)
+          return unless corporation.corporation?
+          return unless corporation.type == :'5-share'
+
+          @log << "#{corporation.name} converts to a 10-share company" unless quiet
+          corporation.type = :'10-share'
+          corporation.float_percent = 20
+
+          shares = @_shares.values.select { |share| share.corporation == corporation }
+          shares.each { |share| share.percent /= 2 }
+          corporation.share_holders.transform_values! { |percent| percent / 2 }
+
+          new_shares = Array.new(5) { |i| Share.new(corporation, percent: 10, index: i + 4) }
+          new_shares.each do |share|
+            add_new_share(share)
+          end
+          new_shares
```

Comment:
I don't see anything that calls `convert!` using this return value, can it be removed?